new-log-viewer/src/utils/downloadDecompressedLogs.ts

```diff
@@ -0,0 +1,23 @@
+interface downloadDecompressedLogsProps {
+    blob: Blob,
+    fileName: string,
+}
+
+/**
+ *
+ * @param blob.blob
+ * @param blob
+ * @param fileName
+ * @param blob.fileName
+ */
+const downloadDecompressedLogs = ({blob, fileName}: downloadDecompressedLogsProps) => {
```

Comment:
Since there are only two arguments needed in this function, can we modify it to accept arguments rather than an object?
```suggestion
const downloadDecompressedLogs = (blob: Blob, fileName: string) => {
```